src/components/screen/SearchUser.tsx

```diff
@@ -1,30 +1,244 @@
-import { NavigationScreenProp } from 'react-navigation';
-import React from 'react';
+import { Animated, FlatList } from 'react-native';
+import { IC_BACK, IC_ICON, IC_SEARCH, IC_SMILE } from '../../utils/Icons';
+import React, { useState } from 'react';
+
+import EmptyListItem from '../shared/EmptyListItem';
+import { User } from '../../types';
+import UserListItem from '../shared/UserListItem';
+import { getString } from '../../../STRINGS';
 import styled from 'styled-components/native';
+import { useStateValue } from '../../contexts';
 
-const Container = styled.View`
+export const userSampleData: User[] = [
```

Comment:
The naming would be better if you use `fakeUsers`. Just to confirm this data is temporary.